.gitlab-ci.yml

```diff
@@ -49,9 +49,10 @@ run-tests:
   stage: test
   image: $CI_REGISTRY_IMAGE/tests-runner:18.09.7-dind
   script:
-    - cd tools/docker
+    - pushd tools/docker
```

Comment:
`pushd` doesn't work because this jobs uses busybox's sh. I don't think it's needed though.
If possible I think `image-pull.sh` should be called from the root dir directly. I don't think we need to keep `image-build.sh`.